sqlglot/dataframe/sql/column.py

```diff
@@ -212,7 +212,12 @@ def sql(self, **kwargs) -> str:
         return self.expression.sql(**{"dialect": SparkSession().dialect, **kwargs})
 
     def alias(self, name: str) -> Column:
-        new_expression = exp.alias_(self.column_expression, name)
+        from sqlglot.dataframe.sql.session import SparkSession
+
+        alias: exp.Expression = sqlglot.maybe_parse(name, dialect=SparkSession().dialect)
+        new_expression = exp.alias_(
+            self.column_expression, alias.this if isinstance(alias, exp.Column) else name
```

Comment:
I check for `exp.Column` here because there were some tests where `name` was `avg(...)`, generated in https://github.com/tobymao/sqlglot/blob/main/sqlglot/dataframe/sql/group.py#L24